### ISSUE TITLE
Unable to locate message source for category 'sprout-forms'.

### ISSUE THREAD

**Issue by ClemensSchneider** [barrelstrength/craft-sprout-seo#170]
Currently I'm struggling getting the lite-version of Sprout-SEO to work.
When trying to create a `Metadata (Sprout SEO)` field (the first), I get the following error:

```
Invalid Configuration – yii\base\InvalidConfigException
Unable to locate message source for category 'sprout-forms'.
```

Caused by the usage of `sprout-forms` as message-source category in `ElementMetadata.php`:

`$this->addError('optimizedTitleField', Craft::t('sprout-forms', 'Upgrade to Sprout SEO PRO to manage multiple Metadata fields.'));`

Unfortunately this is not my main issue as I'm simply not able to create a `Metadata (Sprout SEO)` field...

**Comment by architchandra** [comment]
Facing the exact same issue.
Craft CMS - v3.2.0
Sprout SEO Lite - v4.1.1
Running on Laravel Valet on MacOS Mojave 10.14.5 (18F132)

**Comment by BenParizek** [comment]
Thanks for reporting @ClemensSchneider @architchandra. This should be resolved now in v4.1.3.
